test/integration/provider/registry/policies_controller_test.rb

```diff
@@ -8,7 +8,7 @@ def setup
     @provider = FactoryBot.create(:provider_account)
     login! @provider
     rolling_updates_off
-    rolling_update(:policy_registry, enabled: true)
+    rolling_update(:policy_registry_ui, enabled: true)
```

Comment:
Do we need a test for `policy_registry_ui : false`?

Reply:
Well, if you remove it you will see tests failing. I do not know ...
This is just "renaming" or putting in another level the UI. So if it was not done in the first PR I think it was for a reason